data/sql/updates/pending_db_world/rev_1595118810076078406.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1595118810076078406');
+
+UPDATE `creature_loot_template` SET `Chance` = 40 WHERE `Entry` = 3672 AND `Item` = 5423;
+UPDATE `creature_loot_template` SET `Chance` = 60, `GroupId` = 1 WHERE `Entry` = 3672 AND `Item` = 5422;
+
```

Comment:
just saying, no need for extra new line, only need return carriage (\r not \n)

it's totally trivial but just saying